blas/unit_test/Test_Blas_gesv.hpp

```diff
@@ -13,13 +13,14 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //@HEADER
-// Note: Luc Berger-Vergiat 04/15/21
-//       This test should only be included
-//       in the CUDA backend if TPL MAGMA
-//       has been enabled.
 
-#if !defined(TEST_CUDA_BLAS_CPP) || \
-    (defined(TEST_CUDA_BLAS_CPP) && defined(KOKKOSKERNELS_ENABLE_TPL_MAGMA))
+// only enable this test where KokkosBlas supports gesv:
+// CUDA+MAGMA and HOST+BLAS
+#if (defined(TEST_CUDA_BLAS_CPP) &&                                           \
+     defined(KOKKOSKERNELS_ENABLE_TPL_MAGMA)) ||                              \
+    (defined(KOKKOSKERNELS_ENABLE_TPL_BLAS) &&                                \
+     (defined(TEST_OPENMP_BLAS_CPP) || defined(TEST_OPENMPTARGET_BLAS_CPP) || \
```

Comment:
I would even remove openmptarget here, no one is testing it properly so let's save ourselves an issues that will surely come down the road otherwise.